Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby "2.4.1"
```

Comment:
@Xiaohong-Deng Github shows this branch is up to date with `development` and there are no conflicts.